backstory/src/demo/java/com/google/sps/servlets/GetAnalyzedImagesServlet.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Google LLC
```

Comment:
If we include these files here, I can update them in a new PR to be functional with the front-end.